integrations/docker/images/base/chip-build-minimal/Dockerfile

```diff
@@ -1,27 +1,19 @@
 # This minimal build image is intentionally not based on chip-build
-FROM ubuntu:focal
+FROM ubuntu:24.04
```

Comment:
I openly wonder:  is there a reason to use CalVer (`24.04`) vs the nickname (`noble`)?

I prefer CalVer so +1 from me on the change, but when it was the nickname in the past I wonder if there was a reason why.  (It's certainly possible that there was no particular reason.)

Reply:
Names seem to hide away how old of a release we are talking about. If I would see 20, I would know 4 years old, but focal does not seem as friendly. did this change on purpose.